### DISCUSSION TITLE
Sobre a profissão de dev e seus hobbies

### DISCUSSION THREAD

**Discussion by willianjusten** [frontendbr/forum#778] [Discussão]
Fala pessoal, esse é um post mais para curiosidade, que meio que nasceu de umas conversas que tive com alguns alunos.

Existe muita gente que trabalha como Dev, mas que também tem paixões em outras áreas. Tem também pessoas que eram de outras áreas e que começaram a codar como hobby. Ou seja, acho que no mundo dev é bem comum termos um hobby ou iniciarmos na área como um hobby.

Então, dando início a brincadeira xD

Eu era Químico Industrial e comecei a programar dentro da faculdade e também como Hobby para fazer joguinhos =p

Hoje eu sou programador, mas tenho cada vez me apaixonado mais por fotografia/hiking/camping. Tenho postado umas brincadeiras minhas no [500px](https://500px.com/willianjusten) e no [Unsplash](https://unsplash.com/@willianjusten). 

E você? Começou programação como hobby, se sim, fazia o que antes? E você programador, tem algum hobby?

Acho que várias revelações divertidas vão aparecer aqui. Como alguém dizendo que antes fazia Direito ou que agora é programador, mas gosta mesmo é de fazer cerveja =p

**Comment by galvaodev** [comment]
Muito bom rsrs.
Comecei a programar porque jogava tibia (como muitos acredito), minha vontade era de fazer um site onde pessoal criava sua conta no próprio site e logava no servidor, achava isso de mais, fora o modo de pagamento, e as compras no game. Sim antes trabalhava como pacotador de um mercado, era novo tinha 15 anos.
Foi amor a primeira tentativa (apesar de que era muito difícil querer programar php na época antes de passar pelo front-page (rsrs).

Acredito que Hobby e trabalho se encontra no mesmo objetivo, estudar e ganhar conhecimentos nesse mundo tão maravilhoso!

**Comment by ludioao** [comment]
** história um pouco grandinha **
A minha história já começa lá por meados de 2004 (9 anos). Na verdade queria ser DJ, mas daí tive um pequeno problema... eu era de cidade pequena, como faria a divulgação?

Muito curioso que sou, procurei saber como criar um site, recorri a um amigo do meu pai que já tinha feito alguma coisa com o Frontpage e lancei o primeiro sitezinho. 

Depois disso, precisava de equipamentos... Mas meu pai não quis patrocinar (ainda bem), e desisti da vida de DJ.

Ainda nessa época participava ativamente dos fóruns mais famosos: PHPBrasil, Código Fonte.net (inclusive era moderador do fórum de JavaScript), mas como era criança, mudava de ideia a cada momento, mas sempre gostei de buscar mais sobre programação.

De 2007~2010, eu jogava Tibia, assim como o @jvgm, criei vários OT servers (servidor alternativo - pra não falar pirata), e também os sites pra galera se cadastrar, comprar créditos no jogo, etc.

Em 2011 voltei pra vida Web com outro propósito, queria montar uma rádio online, mas àquela época já não estava mais 'dentro do assunto' e fui estudar um pouco sobre. Fiz algumas coisas com o auxílio do Fireworks (que 'fatiava' o site - sim, aquelas table ridículas)... e lancei a minha rádio online. 

Neste meio tempo, sem muito sucesso (de novo), fui dar uma olhada na parte de Hacking, e qual é o primeiro assunto que um 'noob' vê na internet? SQL Injection... Pois bem, peguei o primeiro site que vi na frente e testei, felizmente, o site tava vulnerável, acessei o admin, mas não fiz nada, só enviei um e-mail pro administrador e ofereci meus serviços. (Não sabia nem por onde começar, e o engraçado é que é cliente meu até hoje).

Em 2013, passei pra Publicidade e Propaganda (privada) e ao mesmo tempo pra Ciência da Computação (federal), optei por fazer CC, e estou até hoje tentando formar) hahaha.

Embora goste da parte de Marketing, acredito que o meu forte é a Programação mesmo. Hahaha

É sempre satisfatório criarmos soluções que possam impactar a vida das pessoas.

**Comment by Pompeu** [comment]
Lutador de Jiujitsu, como hob programador e palestrante.

**Comment by ghost** [comment]
Que issue bacana haha
Eu comecei a programa quando surgiu a necessidade de criar um blog de download, que era febre até pouco tempo atrás, então eu utilizei ferramentas como blogger, ali conheci o html, depois descobrir que existia uma carreira pra aquilo e decidi seguir.

Fora a programação, sou extremamente apaixonado por animação e desenho, já brinquei muito fazendo animações de stickmans no flash, e outros programas, e sempre quis seguir carreira com isso, mas a paixão pelos códigos falou mais alto.

Hoje não faço mais animações por falta de tempo, dedico ele totalmente a estudo envolvendo programação que é profissão, mas também não deixa de ser meu hobby, mas quero voltar sim a desenha e animar algumas histórias hehe

Abraço

**Comment by klarkc** [comment]
Faz tanto tempo q eu programo que nem me lembro direito porque comecei a programar, acho que foi quando eu criava bots para o CS ou criava jogos no RPG Maker. Depois de um tempo comecei a me aventurar nos formulários do Visual Basic que vinha com o office, apenas por curiosidade, daí pulei pro Delphi e a partir daí n teve volta... Eventualmente caí na Web e descobri o Javascript, e cá estou eu :)

Sobre Hobby, adoro tocar e aprender instrumentos, fiz curso de Piano (mas larguei hehe), toco flauta transversal - aprendi sozinho, e bato uns acordes no violão (estou melhorando isso rs). Adoro(va) jogar, mas ultimamente tenho perdido o interesse ( nada me prende por muito tempo :( )

**Comment by willycamargo** [comment]
Desde muito novo sempre quis estar fazendo algo diferente e por criação eu aprendi a dar o melhor de mim em tudo que tivesse que ser feito.

Como sou muito agitado desde novo eu pertubava minha mãe para deixar eu trabalhar com algo (qualquer coisa), mais por interesse do que por dinheiro. Quando eu tinha 13 anos, achando que eu não iria aceitar, ela propôs que eu trabalhasse como ajudante de meu tio que era pedreiro. Eu aceitei e foi uma das melhores coisas que podia ter feito. Aprendi muito sobre construção civil e coisas relacionadas.

Minha mãe é costureira, então por um tempo eu trabalhei como enfestador e cortador de tecidos também, e essa foi outra profissão que me ensinou muita coisa também.

Diante de muitas coisas eu acabei conhecendo desenvolvimento web em um cursinho profissionalizante que comecei a fazer quando tinha 12 anos e acabei com 14/15. Comecei a trabalhar profissionalmente com isso logo que terminei o curso, aos 15 anos e sem registro nem nada (mas logo que fiz 16 fui registrado).

Sinceramente, desde que trabalhei como ajudante de pedreiro eu me considero uma pessoa **apaixonada por criação**. Me sinto extremamente feliz e satisfeito ao criar ou corrigir algo, seja na construção civil, na confeccão de roupas, cozinhando algo, arrumando uma peça de um carro/moto etc..

**Desenvolver sistemas** foi uma forma que encontrei de estar sempre criando ou corrigindo coisas. Isso me satisfaz dia após dia.

Atividades que fiz/faço por interesse e hobby: Muay Thai, corrida, mecânica automotiva, cozinhar, desenhos 2d e 3d.

**Comment by glaucia86** [comment]
O meu interesse por programação surgiu em meados de 2004, quando fazia monitoria da sala de informática, ensinando a comunidade as coisas básicas de informática: Word, Excel, Power Point. Enfim.... depois das aulas de informática, sempre eu pegava o meu disquete do famoso jogo 'Carmen San Diego' (quem nunca aqui já jogou esse jogo....) e dali me despertava algumas curiosidades em como acontecia as coisas e tudo mais.
Antes de ser Dev já fui: Vendedora de Roupas, Vendedora de Seguros, Analista Financeiro e Operadora de Câmbio.
Guardei tudo aquilo para mim.... e em meados de 2011 entrei na Faculdade. E em 12/2013 estava concluindo a Faculdade - que no início eram +- 43 pessoas e concluíram dois homens e eu somente de menina naquele ano. Uma satisfação tamanha que não tem como explicar.
A primeira experiencia veio uma semana depois deu ter concluído a Faculdade.
E desde então, tenho procurado trabalhar e desenvolver projetos pessoais - **por mais que esteja desempregada agora .... :(** Mas, nunca e jamais deixo de fazer o meus commits aqui no GitHub. 
Atualmente, enquanto não estou trabalhando, estou focada em disseminar essa paixão e conhecimento a outras pessoas que queiram entrar nesse lindo mundo que se chama: codar!

Atividades extras que gosto de fazer: viajar, conhecer novas pessoas e escutar as suas histórias, cinéfila, baterista, aprender novos idiomas e amo astronomia! <3

**Comment by wandersonmaceds** [comment]
Wow, legal, eu comecei já programando, fiz curso técnico, faculdade e cheguei até a entrar em uma pós graduação. Mas antes disso, eu já tinha estudado música. No fim das contas, eu sou músico (não profissionalmente apesar de já ter passado em testes para bandas - não levei isso muito a frente) e também programador. 

Apesar de ser um hobby eu já fiz diversos shows de música, mas como profissão mesmo, eu sou ou era  programador. Era por que passei a ser transcritor de um tempo pra cá, então, programação e música ficaram meio como hobbies aparentemente. 

Achei legal essa issue, vou acompanhar :)

**Comment by woliveiras** [comment]
**É O TIBIA FAZENDO HISTÓRIA!**

Meus hobbies são:

* séries e filmes, cinema
* gostava muito de desenhar, porém não ando mais fazendo isso (um dia pretendo voltar)
* adoro andar de longboard e patins

Gostava muito de jogar games online, mas ultimamente não tenho encontrado nenhum que me agrade tanto quanto o Tibia e Word of Warcraft me agradaram num passado não muito distante.

Em todo caso, curto estar com uma galera, por isso sempre tento reunir as pessoas pra fazer coisas em grupo.

Quem for do Grande ABC Paulista e quiser marcar uns rolês na pista de Skate de São Bernardo do Campo, tamo ae!

Quem conhecer algum jogo (RPG, MMO, estratégia e aventura) maneiro, me da um toque. De preferência, hoje em dia, algo que de pra jogar no celular pra eu jogar no busão.

**Comment by mahenrique94** [comment]
Muito bom a issue \o/

Sempre fui da area de informática, trabalhei com meu pai como **Técnico em Informática** onde fazia manutenção em computadores e notebooks, aprendi muito coisa em longos 5 anos, até que resolvi realizar um curso técnico para aprimorar os conhecimentos(primeiro curso), onde tive o primeiro contato com programação(delphi), cara eu amei aquilo \o/, realmente foi amor a primeira vista.

Até que fui indicado para entrar no suporte da empresa que estou desenvolvendo hoje, em alguns meses estudante, sai do suporte e comecei a corda(onde conheci a web e java), rapaz o que já era amor virou perfeição kk.

Sobre hobbys, gostava bastante de jogar CS(atualmente dei uma parada), porém, amo jogar futebol(pelo menos tentar kk).

Nos tempos vagos gosto bastante de ler, fazer cursos, sair com os amigos, jogar um truco kk e recentemente estou amando o desenvolvimento de games.

Abraçoooos

**Comment by AmandaFonseca** [comment]
Eu sempre gostei de computador e tecnologia fiz um curso de manutenção de computadores mas tive que parar pq era muito longe da minha casa e minha mãe era muito medrosa. Eu não queria trabalhar com manutenção de computadores mas acabei tendo que trabalhar em outros lugares com outras coisas tipo numa padaria,  depois numa loja, telemarketing ou ajudando minha mãe que é costureira (@willycamargo super me identifiquei com a sua história) até que me aconselharam fazer um curso na área da saúde pois minha irmã enfermeira e podia me ajudar a conseguir trabalho.
Acabei ganhando um bolsa para fazer o curso técnico de farmácia, trabalhei num hospital mas não gostei descobri que o irmão do meu cunhado é analista de sistemas  e eu fui pesquisando mas  sobre a área e gostei fiz o curso de analista de sistemas e estou no meu primeiro emprego na área.

Hoje em dia trabalho numa empresa de T.I ainda não 100% como desenvolvedora mas as vezes me passam uns projetos.

Como  hobbie eu gosto de estudar novas tecnologias, assistir filmes e séries, gosto de barzinho e as vezes frequento motoclubes que tem show de covers de bandas de rock antigas. 

Geralmente quando eu vou nesse rolê eu sou sempre a mais nova do grupo mas eu adoro kkkk

**Comment by felipefialho** [comment]
Curti, 😄 

Alguns dos meus hobbies principais são:

## Futebol

Assistindo, jogando, não importa, futebol é das coisas que mais gosto de fazer na vida. Sou realmente transportado para outro mundo e relaxo muito quando estou assistindo ou jogando futebol.

Sou Corinthiano fanático e não perco nenhum jogo, mas em geral paro para assistir qualquer jogo que esteja passando. Além disso jogo futebol todo domingo faz 10 anos e modestia a parte... sou atacante dos bons (tinha vários videos no Youtube, pena que apagamos), haha.

## Bar

Adoro bar. Sentar com meus amigos e ficar horas jogando conversa fora enquanto tomo uma cerveja gelada. É outra coisa que me relaxa bastante e tento manter uma frequencia.

## Musculação

Faço musculação desde 2007, praticamente sem pausas. Na verdade já gostei muito mais do que gosto hoje, minha frequência já chegou a ser 5 ou 6 vezes por semana, com alimentação e rotina toda focada para ganho muscular.

Hoje treino 3 vezes por semana.

## Assistir series

Já fui fissurado em filmes, [devo ter visto mais de 1000](http://www.cineplayers.com/usuario/Paranoiko/filmes), mas hoje em dia não tenho tanta paciência para ficar 3h focado em um filme. Vejo bem menos.

As series acabaram preenchendo esse espaço. Mesmo em alguns casos ficando as mesmas 3h assistindo, posso parar entre os episódios.

____

Poderia colocar coisas como ir em restaurantes, ou viajar. Mas isso é muito obvio, todo mundo gosta, hehe.

**Comment by n3pu** [comment]
@daniilomello e @willycamargo resumiram um pouco o que aconteceu comigo. Pai mestre de obras, tive que ajudar e trabalhar por alguns momentos com ele, hoje sei me virar graças a isso. As mãos um pouco macias de hoje são um pouco diferentes de antes mais calejadas. Mãe costureira, sempre me mostrando seus resultados e artes. Eu desenhava bastante à mão, uma vez fiz um curso técnico que me levou a design gráfico e programação. Fiquei na corda bamba. O desenho me chamava mais atenção, mas a facilidade que eu tinha com números me levou a Java. Bem... acabei chutando o balde e fiz escola de Efeitos Visuais. Mais uma vez o futuro me pregou uma peça, ganhei uma bolsa para Análise e Desenvolvimento de Sistemas. Mas como conciliar design com programação? Ai parei aqui =) não é impossível. Certo que ainda engatinho em programação (JavaScript), mas consigo fazer bastante coisa e acompanhar os projetos por inteiro.

Hobby: **Desenho, Pintura, Trabalhos manuais (criar, ajeitar, modificar), Ciclista urbano, Ciclo viagens, Moto viagens, Séries.**

_Desejo: Comer e respirar sem engordar tanto_

**Comment by mosesmeirelles** [comment]
Que legal essa thread, me identifiquei com muita gente! Meu Pai pedreiro e Mãe confeiteira, ela acabava me pedindo para ajudar com os doces no fim de semana, foi ali que aprendi a desenhar olhos e mexer com pasta americana hahaha. 

Esse pé na arte e ainda apesar de ser um jogador casual, as cinematics de World of Warcraft e Diablo, me despertaram um desejo enorme de estudar desenho (que acabei fazendo curso por um ano só) e trabalhar com design ou concept art, apesar disso, aqui por morar em cidade pequena, não havia nada nessa área além de Publicidade e Propaganda, mas infelizmente era privada e eu e meus pais não tinhamos condições de manter. Por influência de um amigo que me ensinou programação, optei por ingressar no curso público de Análise e Desenvolvimento de Sistemas. Cheguei até a trabalhar em uma agência pequena aqui da cidade, foi onde aprendi **Photoshop e Illustrator**, mas cada vez mais fui indo para o trabalho da programação e Front End por existirem algumas vagas em empresas daqui e assim me mantenho na programação até agora.

Esse mês voltei a estudar **desenho** e **pintura**, e desejo ainda conseguir montar um portfólio disso, mesmo sem ser um objetivo profissional em si. Decidi aprender a cozinhar de verdade, então comecei a me aventurar na cozinha também. Fora isso, eu arranho algumas notas na guitarra/violão, mas não me peçam para tocar nenhum riff porque eu não sei nenhum hahaha.

**Comment by rubenmarcus** [comment]
bacana essa issue.

Eu sou Artista de Rua ( street artist ) - www.psy21.love - https://www.instagram.com/psy21.love
e fotografo de rua / culturas / humanitário - www.rubenluz.com -  https://www.instagram.com/rubenluz

**Comment by yanmagale** [comment]
Muito Show 😄 

Alguns hobbies que tenho:

- Futebol (quando posso, jogo)
- F1
- Assistir Gameplays de F1
- Conhecer lugares novos
- Ver Filmes
- Sair com amigos/família

**Comment by felipefialho** [comment]
E esse ano tem um dos meus maiores hobbies: COPA DO MUNDO

**Comment by kvnol** [comment]
Acho que, para qualquer profissão, ter hobbies são essenciais, para evitar o stress diário. Alguns dos meus:

1. Música (violão, guitarra, percussão, etc. - já está deixando de virar hobbie porque já faço freelas como músico, mas beleza);
2. Musculação;
3. Artes marciais (muay thai e boxe);
4. Futebol;

**Comment by florbraz** [comment]
No começo dos anos 00, nos **fóruns de anime, jogos e Pump it Up**, começou meu interesse por Photoshop (para aprender a fazer aquelas assinaturas muito invocadas hahahah). Aprendi bastante coisa e meu primeiro emprego foi como designer gráfica. Acabei também ajudando nos designs pra web de alguns sites. Acabou sendo natural querer descobrir como que eu faria aqueles designs funcionarem. E cá estamos hoje. :)

Agora, mais tiazinha, rs, meus hobbies são basicamente:
- Cozinhar;
- Plantar;
- Sustentabilidade / evitar desperdícios;

_(coincidentemente com me chamar **"Flora"**. hahahah)_
